README.rst

```diff
@@ -68,3 +73,10 @@ You can also use tox to run tests against Python 3.6 and 3.7
 
     $ . .venv/bin/activate
     $ tox
+
+KNOWN ISSUES
+------------
+
+After running :code:`$ make dev-run` integration test (and possibly all connection to ZUUL) will fail.
+
+Current fix is to add read permissions for file :code:`/etc/zuul/zuul.conf` and manually start service :code:`zuul-scheduler` in virtual machine.
```

Comment:
start service :code:`zuul-scheduler` -> start :code:`zuul-scheduler` service
in virtual machine -> inside 'vagrant'